presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add documentation for Glue Catalog support in Hive. :doc:`/connector/hive`.
+* Add iceberg connector.
+* Add support to fragment result caching for unnest.
+* Add :func:`poisson_cdf` and :func:`inverse_poisson_cdf` functions.
+* Add memory tracking in ``TableFinishOperator`` which can be enabled by setting the ``table-finish-operator-memory-tracking-enabled`` configuration property to ``true``.
+* Remove spilling strategy ``PER_QUERY_MEMORY_LIMIT`` and instead add configuration property ``experimental.query-limit-spill-enabled`` and session property ``query_limit_spill_enabled``.  When this property is set to ``true``, and the spill strategy is not ``PER_TASK_MEMORY_THRESHOLD``, then we will spill whenever a query uses more than the per-node total memory limit in combined revocable and non-revocable memory, in addition to whenever the memory pool exceeds the spill threshold.  This fixes an issue where using the ``PER_QUERY_MEMORY_LIMIT`` spilling strategy could prevent the oom killer from running when the memory pool was full.  The issue is still present for the ``PER_TASK_MEMORY_THRESHOLD`` spilling strategy.
+
+Hive Connector Changes
+______________________
+* Add max error retry config option to glue client. Defaults to 10. :doc:`/connector/hive`.
+* Add support for Glue endpoint URL :doc:`/connector/hive`.
+* Add intelligent tiering storage class. The S3 storage class to use when writing the data. ``STANDARD`` and ``INTELLIGENT_TIERING`` storage classes are supported. Default storage class is ``STANDARD`` :doc:`/connector/hive`.
+
+Hive Changes
+____________
+* Add support for MaxResults on Glue Hive Metastore.
+* Add support for bucket sort order in Glue when creating or updating a table or partition.
+* Add support for partition cache validation. This can be enabled by setting ``hive.partition-cache-validation-percentage`` configuration parameter.
+* Add support for partition schema evolution for parquet.
```

Comment:
Can we add more detail in the release note here about what we mean by schema evolution? Does it mean you can query tables that have had columns added/deleted? column types changed?

Reply:
@imjalpreet @zhenxiao - could you help provide more details

Reply:
support add/delete/replace columns in partition schema
@imjalpreet could you please add more details?

Reply:
We can write something like:

```
Add support for allowing to match columns between table and partition schemas by names when the configuration property ``hive.parquet.use-column-names`` or the hive catalog session property ``parquet_use_column_names`` is set to true. By default they are mapped by index. 
```

Reply:
If we want to add more: 

This allows schema evolution between table and partition in addition to schema evolution between the table/partition and file. Columns can be re-ordered, added or dropped between partition and table schemas.